example/profile.php

```diff
@@ -1,32 +1,42 @@
 <?php
-//require_once __DIR__ . '/../src/boot.php';
-require_once './vendor/autoload.php';//uses composer
-
+require_once './vendor/autoload.php';//make sure you run composer update inside the example folder before trying them
```

Comment:
I feel like these comments should be put into a README.md file in the `/example` folder, or be added to the main Readme file. Here's an example of how the Ruby SDK does it: https://github.com/getyoti/ruby#running-the-examples. Ideally, we should stick to a comment coding standard. I prefer the `[space]//[space][comment starting with uppercase letter]`